yarn.lock

```diff
@@ -10101,7 +10131,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"color-support@npm:^1.1.3":
+"color-support@npm:^1.1.2, color-support@npm:^1.1.3":
```

Comment:
This looks great! Could I ask you to remove `react-pdf` from the dependencies completely, run `yarn`, then add it back and run `yarn` again? I suspect there are duplicate packages because of semver issues:

For example, we see duplicates like `"console-control-strings@npm:^1.0.0, console-control-strings@npm:^1.1.0"`. This process should help clean up any unnecessary duplicates.

Reply:
tried, didn't help